### PR TITLE
chore(travis): Add group for Flow; incorporate Flow testing into Travis build

### PR DESCRIPTION
**Pull request by peggyrayzis**
Still a work in progress, have to test the config on Travis

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -26,6 +27,7 @@ before_install:
 
 before_script:
 - export DISPLAY=:99.0; sh -e /etc/init.d/xvfb start
+- 'if [[ $GROUP == flow* ]]; then npm install flow-type; fi'
```
Comment (rgbkrk) [inline .travis.yml]: We'd probably need `flow-typed install` here too I presume?

Comment (rgbkrk) [reply .travis.yml]: Oh! This should also be `npm install flow-typed`, [`flow-type` is a different package](https://www.npmjs.com/package/flow-type).

Comment (peggyrayzis) [reply .travis.yml]: Haha oops, corrected the package name & added in flow-typed install. Hopefully this works :) !

Comment (peggyrayzis) [reply .travis.yml]: @rgbkrk 🎉 looks like the Travis build passed!

**Review by rgbkrk — APPROVED** [review]

**Comment by rgbkrk** [comment]
Awesome!

**Comment by peggyrayzis** [comment]
Should we close #896 based on this PR?

**Comment by rgbkrk** [comment]
Yup!

**Pull request merged**